aclk/aclk_lws_wss_client.c

```diff
@@ -0,0 +1,340 @@
+#include "aclk_lws_wss_client.h"
+
+#include "libnetdata/libnetdata.h"
+
+#define UNUSED(x) (void)(x)
```

Comment:
Please, move this definition to a global file, my suggestion `libnetdata/libnetdata.h`, because it is very useful!
Thank you to bring it and congratulations! :)

Reply:
resolved in #7988